nsiqcppstyle_exe.py

```diff
@@ -174,40 +220,12 @@ def main(argv=None):
 
         updateNsiqCppStyle = False
         for o, a in opts:
-            if o in ("-h", "--help"):
-                print(title)
-                Usage()
-            elif o in ("-r", "--list-rules"):
-                ShowRuleList()
-            elif o == "-o":
-                outputPath = a.strip().replace('"', "")
-            elif o == "--update":
+            if o == "--update":
```

Comment:
The entire --update feature should be removed.  The original author of NsiqCppStyle project, JunHo Yoon, added the feature to enable updating the source code from his company's server.  Here is the code in nsiqcppstyle_exe.py:

<snippet>
def Update():
    console.Out.Error("Development in progress. Please check manually")
    """
    updateagent.agent.Update("http: //nsiqcppstyle.nsiq.nhncorp.com/update",
                             version)
    """
</snippet>

NHN Corp. was the company he worked for.  Today developers should use "git pull" to update their clones.

I recommend removing all traces of "--update".

Reply:
Point. I was planning on removing it, just not in this PR

Reply:
I don't see any code change.  You should leave a comment on what you did or intend to do.

Reply:
You should be able to observe the latest change. Maybe my rebasing had an impact?

Reply:
Shouldn't I see all changes via the PR changes web page?  Do I need to pull anything to see the changes?

Reply:
I think you are looking at an older change. I can clearly see the update
![image](https://github.com/kunaltyagi/nsiqcppstyle/assets/2657068/de173a2c-7424-4e3d-9626-78d5dd37aa7e)